app/customer.py

```diff
@@ -0,0 +1,48 @@
+from math import sqrt
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: dict) -> None:
+
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    def calculate_trip_cost(self, shop: dict, fuel_price: float) -> float:
+        distance_to_shop = self.calculate_distance(
+            self.location,
+            shop.location
+        )
+
+        total_distance = distance_to_shop * 2
+        fuel_cost = ((total_distance / 100)
+                     * self.car.fuel_consumption

```

Comment:
Since `self.car` should be an instance of the `Car` class, you can access `fuel_consumption` directly as `self.car.fuel_consumption`.